### DISCUSSION TITLE
Archive folder

### DISCUSSION THREAD

**Discussion by nikuda** [opengaming/osgameclones#1385] [Ideas]
We should create an `archive` folder with the same structure as `games`. When a game server/source becomes unreachable and there's no backup anywhere instead of deleting the game entry we move it to archive. 

It'd be useful to keep 'lost' games in archives incase another source is later found, or somebody wants to dig around for the source.

**Comment by tzizi** [comment]
How about an `archived` property instead? I believe it would be a bit less work for the same result.

**Comment by pehsa** [comment]
I think there is no meaningful benefit to leave the unreachable games.
If we keep them the webpage size will increase but no useful information is stored because the visitors don't able to download or locate those games.

Plus if we are lucky the Internet Archive is storing the page.
